_episodes/16-parallel.md

```diff
@@ -1,28 +1,36 @@
 ---
 title: "Running a parallel job"
 teaching: 30
-exercises: 30
+exercises: 60
 questions:
 - "How do we execute a task in parallel?"
+- "What benefits arise from parallel execution?"
+- "What are the limits of gains from execution in parallel?"
 objectives:
-- "Understand how to run a parallel job on a cluster."
+- "Construct a program that can execute in parallel."
+- "Prepare a job submission script for the parallel executable."
+- "Launch jobs with increasing degrees of parallel execution."
+- "Record and summarize the timing and accuracy of the results."
```

Comment:
```suggestion
- "Launch jobs with parallel execution."
- "Record and summarize the timing and accuracy of jobs."
- "Describe the relationship between job parallelism and performance."
```

I think these are three separate skills, and should be captured in three different objectives: you are teaching learners how to 1. run a  job in parallel, 2. how to access and parse information about the performance of a job, and 3. how to translate that into an understanding of the different ways that parallel execution can effect job performance.